01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,14 @@
+(1..20).each do |i|
```

Comment:
```suggestion
(1..20).each do |n|
```

大きな問題ではないですが数字をnumber(num)と呼ぶなら、numberの`n`にした方が一貫性があっていいかもです